cloud/aws/services/ec2/securitygroups.go

```diff
@@ -17,34 +17,12 @@ import (
 	"fmt"
 
 	"github.com/aws/aws-sdk-go/aws"
-	"github.com/aws/aws-sdk-go/aws/awserr"
 	"github.com/aws/aws-sdk-go/service/ec2"
 	"github.com/golang/glog"
 	"github.com/pkg/errors"
 	"sigs.k8s.io/cluster-api-provider-aws/cloud/aws/providerconfig/v1alpha1"
 )
 
-const (
-	portSSH             = 22
-	descSSH             = "SSH"
-	protSSH             = "tcp"
-	portKubeAPI         = 6443
-	protKubeAPI         = "tcp"
-	descKubeAPI         = "Kubernetes API"
-	portKubelet         = 10250
-	descKubelet         = "Kubelet API"
-	protKubelet         = "tcp"
-	portEtcd            = 2379
-	descEtcd            = "etcd"
-	protEtcd            = "tcp"
-	portEtcdPeer        = 2380
-	descEtcdPeer        = "etcd peer"
-	descNodePortSvc     = "Node Port Services"
-	protNodePortSvc     = "tcp"
-	portFromNodePortSvc = 30000
-	portToNodePortSvc   = 32767
-)
```

Comment:
@randomvariable I removed these consts and generalized some of the repetition in other parts (e.g. "tcp" in types.go)